sims/vsim/Makefile

```diff
@@ -86,6 +86,12 @@ $(sim_debug) : $(sim_vsrcs) $(sim_dotf)
 	rm -rf csrc && $(VCS) $(VCS_OPTS) -o $@ \
 	+define+DEBUG -debug_pp
 
+#########################################################################################
+# helper rules to run simulator with debug
+#########################################################################################
+run-binary-debug: $(sim_debug)
+	$(sim_debug) $(PERMISSIVEON) $(SIM_FLAGS) +vcdplusfile=$(long_name).vpd $(PERMISSIVEOFF) $(BINARY)
```

Comment:
I would prefer if `$(long_name).vpd` was `$(BINARY).vpd`.

Also stderr should be piped ` 2> $(BINARY).out`. The output from `+verbose` goes to stderr.

Reply:
How about something like `BINARY.LONG_NAME.out/vpd`. This matters because if you have multiple sims in the same folder and are running different configs, then the same file would conflict.

Reply:
Sure. I didn't think of that.